# Swine-microbiota

Clinical complications during respiratory viral infections commonly contribute to the dysregulation of the microbiome, altering the balance and relationships between different bacterial communities, and potentially favoring pathogenesis. To study the effect of influenza infection on these bacterial communities, in Chapter 3, we analyzed the lung microbiota of Influenza infected pigs, which serve as a model for humans, due to the limitations in human lung sampling. Using Oxford Nanopore's long-read 16S (ONT) sequencing on samples from multiple Spanish regions, we found a greater diversity and abundance of bacterial genera in infected animals compared to healthy ones. Notably, Glaesserella was present in 60% of infected samples and was frequently dominant. Other pathogenic bacteria, such as Pasteurella, Staphylococcus, Mycoplasma, and Fusobacterium, also showed significant associations with infection. Clustering methods allowed us to distinguish healthy from infected animals, identifying specific bacterial groups predictive of infection status. This study suggests that influenza reshapes the respiratory microbiota, promoting potential pathogens that increase secondary infection risk. Moreover, we developed a streamlined ONT-based pipeline for rapid, efficient bacterial profiling in swine models, enhancing our ability to study respiratory microbiota in disease contexts. 

## ðŸŽ¯ Objectives

In this study, we employed Oxford Nanopore sequencing technology (ONT) to investigate the pulmonary microbiome of pigs infected with Influenza virus from different locations in Spain with a targeted Next Generation Sequencing (NGS) metagenomic approach, by generating long-read 16S rRNA sequences. This analysis provides a more comprehensive assessment of bacterial diversity and composition in the context of Influenza infection and pathogenesis, offering insights into potential frameworks for disease understanding. 


 ## ðŸ”¬ Metodology

Clinical samples (lung, trachea, and nasal exudates) from healthy and sick pigs with respiratory symptoms were provided by farm veterinarians from  main pig-producing regions of Spain. A preliminary diagnostic test of  Influenza A-specific RT-PCR was performed. Positive samples were confirmed by RT-PCR in our laboratory and processed for Influenza virus isolation and whole genome sequencing. Extracted DNA from healthy and infected samples was used to amplify the 16S rRNA gene of the bacteria present in the sample using the 16S Barcoding kit (SQK-16S024) from Oxford Nanopore Technologies (ONT). After amplification, DNA was purified and sequenced using the MinION Mk1b ONT. 
After OTUs abundance obtention, the microbial composition of samples and groups was analyzed and plotted using R, and a comparison between groups was performed by five different methods: Wilcox rank sum test (stats R package), biserial correlation function based on sensitivity/sensibility (multipatt function from indicspecies R package), Pearson correlation based on presence (multipatt), LinDa method (microbiomeStat R package), and Boruta (Boruta R package). The significant bacteria from all methods were further selected for the generation of a prediction model using a Flexible Discriminant Analysis (FDA) using the mda R package. All methods were compared based on their discrimination abilities and the Z scores obtained.

## Main conclusions

The complex inflammatory regulation of the host-respiratory and immune systems during influenza determines the severity of the disease. While much attention has been given to individual pathogens, such as different Influenza viruses and the bacteria associated with secondary infections, this traditional view may be overly simplistic. The data suggest that Kock's postulates no longer fully explain the complexity of the disease caused by some infectious agents. The role of different bacteria associated with infections by Influenza viruses requires further investigation. Alternative dysbiosis contexts can lead to similar disease severities, requiring different therapeutic approaches. Both descriptive and functional microbiota assays are essential to better understand disease etiology and develop more effective treatments.

> **Contacto:** Javier Arranz Herrero | [LinkedIn](https://www.linkedin.com) | [Email](mailto:j.arranzherrero@gmail.com)
